### PR TITLE
Dataup 290 param label

### PR DESCRIPTION
**Pull request by leia-sefkin**
# Description of PR purpose/changes

* This is a fix on DATAUP-290 - to have the parameter labels display above the fields rather than next to. Uses the changes from the [file paths PR](https://github.com/kbase/narrative/pull/1967) to achieve this.

Screen Shot 2020-11-22 at 4 22 41 PM

### CODE REVIEW

File: kbase-extension/scss/partials/_bulkImportCell.scss [hunk 1]
```diff
@@ -221,6 +221,15 @@ $button-outdated: rgb(247, 155, 34);
...
     margin: 3px;
   }
+
+  &__input_control select {
+    margin-left: 0;
+  }
+
+  &__cell_label {
+    margin-bottom: 0;
+    padding-top: 8px;
```
Comment (leia-sefkin) [inline kbase-extension/scss/partials/_bulkImportCell.scss:231]: i'm not sold on this approach, but it was the fastest way to achieve more reasonable spacing of the fields and labels ahead of the demo

File: kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js [hunk 2]
```diff
@@ -388,14 +388,17 @@ define([
             });
 
             const layout = orderedParams.map(function (parameterId) {
-                let id = html.genId();
+                const elementId = html.genId();
```
Comment (leia-sefkin) [inline kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js:391]: just changed variable name so it's more clear parameter id vs. element id

File: kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js [hunk 3]
```diff
@@ -388,14 +388,17 @@ define([
...
+                const advanced = paramMap[parameterId].ui.advanced ? parameterId : false;
```
Comment (leia-sefkin) [inline kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js:392]: it really was this simple in the end tho

Comment (eamahanna) [reply kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js:392]: Simple solution are the best 🎉

**Review by ialarmedalien — APPROVED** [review]
LGTM. We can always do clean up after the demo.

**Review by eamahanna — APPROVED** [review]

**Pull request merged**